.asf.yaml

```diff
@@ -0,0 +1,27 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+github:
+  description: "The mission of Apache Ambari is the creation and maintenance of software related to Hadoop cluster management"
```

Comment:
Can we have like "Apache Ambari simplifies provisioning, managing, and monitoring of Apache
Hadoop clusters."  as that's in our project board resolution.?

Reply:
This was taken from reporter.apache.org.

The statement "Apache Ambari simplifies provisioning, managing, and monitoring of Apache Hadoop clusters." was in the board report from the previous life.

I can change it, sure.

Reply:
ok